source/09-audit/03-timelines.rst

```diff
@@ -0,0 +1,59 @@
+.. The COUNTER Code of Practice Release 5 © 2017-2023 by COUNTER
+   is licensed under CC BY-SA 4.0. To view a copy of this license,
+   visit https://creativecommons.org/licenses/by-sa/4.0/
+
+Audit Timelines
+---------------
+
+An audit is valid for twelve months from the date of seeding: that is, an audit seeded in April 2025 will be valid until the end of March 2026.
+
+The audit process can be lengthy and report providers should take this into consideration when planning for audits. A typical timeline is shown in table 9.a below.
+
+Table 9.a (below): Audit Timelines
+
+.. only:: latex
+
+   .. tabularcolumns:: |>{\raggedright\arraybackslash}\Y{0.13}|>{\raggedright\arraybackslash}\Y{0.17}|>{\parskip=\tparskip}\Y{0.37}|>{\raggedright\arraybackslash}\Y{0.33}|
+
+.. list-table::
+   :class: longtable
+   :widths: 38 38 24
+   :header-rows: 1
+
+   * - Process
+     - Usual Duration
+     - Example
+
+   * - Seeding
+     - 1 calendar month
+     - February
+
+   * - Usage processed, reports available
+     - Up to 28 days from end of seeding month
+     - March
+
+   * - Report reconciliation
+     - 4 to 6 weeks
+     - April
+
+   * - Fixes after interim report
+     - Up to 3 months
+     - May to July
+
+   * - Report reconciliation after fixes
+     - 1 month
+     - August
+
+\*Report providers may request extensions to the usual timelines under the following circumstances.
+
+* **Delayed seeding** - where a report provider is able to demonstrate that they have run the COUNTER Validation Tool and are in the process of fixing problems identified by the Tool before commencing an audit, they may apply for up to three months delay in the seeding process.
```

Comment:
"the tool" (lower case, as in other places where this term is used for the Validation Tool) or "the Validation Tool"